### PR TITLE
Update property names

### PR DESCRIPTION
**Pull request by svaruag**
# PR into Azure/azureml-examples

## Checklist

I have:

- [x] read and followed the contributing guidelines

## Changes

- Update property names (timeout => timeout_minutes, trial_timeout => trial_timeout_minutes, [allowed/blocked]_models=[allowed/blocked]_training_algorithms
- Update JobInput => Input

fixes #

### CODE REVIEW

File: sdk/jobs/automl-standalone-jobs/automl-classification-task-bankmarketing/automl-classification-task-bankmarketing-mlflow.ipynb [hunk 1]
```diff
@@ -241,8 +241,8 @@
     "\n",
     "# Limits are all optional\n",
     "classification_job.set_limits(\n",
-    "    timeout=600,  # timeout_minutes\n",
-    "    trial_timeout=20,  # trial_timeout_minutes\n",
+    "    timeout_minutes=600,  # timeout_minutes\n",
+    "    trial_timeout_minutes=20,  # trial_timeout_minutes\n",
```
Comment (msp8955) [inline sdk/jobs/automl-standalone-jobs/automl-classification-task-bankmarketing/automl-classification-task-bankmarketing-mlflow.ipynb:245]: remove comments

File: sdk/jobs/automl-standalone-jobs/automl-classification-task-bankmarketing/automl-classification-task-bankmarketing-mlflow.ipynb [hunk 2]
```diff
@@ -158,7 +158,7 @@
     "- `target_column_name` - The name of the column to target for predictions. It must always be specified. This parameter is applicable to 'training_data', 'validation_data' and 'test_data'.\n",
     "- `primary_metric` - The metric that AutoML will optimize for Classification model selection.\n",
     "- `training_data` - The data to be used for training. It should contain both training feature columns and a target column. Optionally, this data can be split for segregating a validation or test dataset. \n",
-    "You can use a registered MLTable in the workspace using the format '<mltable_name>:<version>' OR you can use a local file or folder as a MLTable. For e.g JobInput(mltable='my_mltable:1') OR JobInput(mltable=MLTable(local_path=\"./data\"))\n",
+    "You can use a registered MLTable in the workspace using the format '<mltable_name>:<version>' OR you can use a local file or folder as a MLTable. For e.g Input(type=\"mltable\", path='my_mltable:1') OR Input(type=\"mltable\", path=local_path=\"./data\")\n",
```
Comment (msp8955) [inline sdk/jobs/automl-standalone-jobs/automl-classification-task-bankmarketing/automl-classification-task-bankmarketing-mlflow.ipynb:161]: should we put as type=AssetTypes.MLTABLE? or string value?

File: sdk/jobs/automl-standalone-jobs/automl-classification-task-bankmarketing/automl-classification-task-bankmarketing.ipynb [hunk 3]
```diff
@@ -230,8 +230,8 @@
     "\n",
     "# Limits are all optional\n",
     "classification_job.set_limits(\n",
-    "    timeout=600,  # timeout_minutes\n",
-    "    trial_timeout=20,  # trial_timeout_minutes\n",
+    "    timeout_minutes=600,  # timeout_minutes\n",
+    "    trial_timeout_minutes=20,  # trial_timeout_minutes\n",
```
Comment (msp8955) [inline sdk/jobs/automl-standalone-jobs/automl-classification-task-bankmarketing/automl-classification-task-bankmarketing.ipynb:234]: Remove these comments

File: sdk/jobs/automl-standalone-jobs/automl-forecasting-task-energy-demand/automl-forecasting-task-energy-demand-advanced-mlflow.ipynb [hunk 4]
```diff
@@ -219,7 +220,7 @@
     "### Using lags and rolling window features\n",
     "This training is also using the **target lags**, that is the previous values of the target variables, meaning the prediction uses a horizon. We therefore must still specify the `forecast_horizon` that the model will learn to forecast. The `target_lags` keyword specifies how far back we will construct the lags of the target variable, and the `target_rolling_window_size` specifies the size of the rolling window over which we will generate the `max`, `min` and `sum` features.\n",
     "\n",
-    "This notebook uses the .set_training(blocked_models=) parameter to exclude some models that take a longer time to train on this dataset.  You can choose to remove models from the blocked_models list but you may need to increase the iteration_timeout_minutes parameter value to get results."
+    "This notebook uses the .set_training(blocked_training_algorithms=) parameter to exclude some models that take a longer time to train on this dataset.  You can choose to remove models from the blocked_training_algorithms list but you may need to increase the iteration_timeout_minutes parameter value to get results."
```
Comment (msp8955) [inline sdk/jobs/automl-standalone-jobs/automl-forecasting-task-energy-demand/automl-forecasting-task-energy-demand-advanced-mlflow.ipynb:223]: .set_training(blocked_training_algorithms=) -> remove '=' at the end?

iteration_timeout_minutes -> trial_timeout_minutes

(also in automl-forecasting-task-energy-demand-advanced.ipynb)

File: sdk/jobs/automl-standalone-jobs/automl-classification-task-bankmarketing/automl-classification-task-bankmarketing.ipynb [hunk 5]
```diff
@@ -323,13 +323,16 @@
   }
  ],
  "metadata": {
+  "interpreter": {
+   "hash": "69daf304a5afb4d75ee1bae08a753430bc753de517d51cf450bbf47abb30c09c"
+  },
```
Comment (msp8955) [inline sdk/jobs/automl-standalone-jobs/automl-classification-task-bankmarketing/automl-classification-task-bankmarketing.ipynb:328]: are these expected change?

Comment (svaruag) [reply sdk/jobs/automl-standalone-jobs/automl-classification-task-bankmarketing/automl-classification-task-bankmarketing.ipynb:328]: I did run the jupyter clear-output command on these files, let me remove this manually (don't think it has any effect though, it'll probably be updated with another interpreter hash when the notebook is attached to a kernel)

**Review by PhaniShekhar — APPROVED** [review]

**Pull request merged**